src/daemon-node.js

```diff
@@ -117,13 +118,16 @@ class Node {
       initOpts = {}
     }
 
-    const keySize = initOpts.keysize || 2048
-
     if (initOpts.directory && initOpts.directory !== this.path) {
       this.path = initOpts.directory
     }
 
-    run(this, ['init', '-b', keySize], { env: this.env }, (err, result) => {
+    const args = ['init', '-b', initOpts.keysize || 2048]
+    if (initOpts.pass) {
```

Comment:
@richardschneider this should already happen since args just get forwarded to the daemon.